libs/wingsdk/src/target-tf-aws/counter.ts

```diff
@@ -25,6 +25,16 @@ const NAME_OPTS: NameOptions = {
  * @inflight `@winglang/sdk.cloud.ICounterClient`
  */
 export class Counter extends cloud.Counter implements IAwsCounter {
+  /** @internal */
+  public static _toInflightType(): string {
+    return core.InflightClient.forType(
+      __filename
+        .replace("target-tf-aws", "shared-aws")
+        .replace("counter", "counter.inflight"),
+      "CounterClient"
+    );
+  }
```

Comment:
In theory the function "toLiftableModuleType" should handle this / allow us to avoid having to implement _toInflightType everywhere, but I think the compiler is generating bad require paths or something like that. Needs more investigation...